### PR TITLE
Add initial jetbrains docs

### PR DESCRIPTION
**Pull request by taylorsperry**
Quick, initial update to mention the Cody Jetbrains extension. 

## Test plan

docs update n/a

### CODE REVIEW

**Review by slimsag — APPROVED** [review]

File: doc/app/index.md [hunk 1]
```diff
@@ -40,9 +40,9 @@ Check the [latest release](https://github.com/sourcegraph/sourcegraph/releases/t
...
 
-If you use VS Code, we recommend you follow the steps to download the VS Code extension, which enables Cody within your editor. (If you installed the extension before you downloaded the app, you'll see a prompt in your editor to download the app.) Cody in VS Code will then talk to your Sourcegraph app to answer questions.
+If you use VS Code or IntelliJ, we recommend you follow the steps to download the extension, which enables Cody within your editor. (If you installed the extension before you downloaded the app, you'll see a prompt in your editor to download the app.) Cody in your editor will then talk to your Sourcegraph app to answer questions.
```
Comment (vdavid) [inline doc/app/index.md:43]: Not just IntelliJ. We support all JetBrains IDEs. Maybe use one of these phrases
- a JetBrains IDE
- IntelliJ or any IDE from the JetBrains family
- IntelliJ IDEA, PyCharm, GoLand, or any other JetBrains IDE

File: doc/cody/explanations/installing_jetbrains.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Installing Cody in Jetbrains (Experimental)
+
+Please visit the Jetbrains Marketplace for [installation instructions](https://plugins.jetbrains.com/plugin/9682-sourcegraph)
```
Comment (vdavid) [inline doc/cody/explanations/installing_jetbrains.md:3]: JetBrains is capital B

```suggestion
Please visit the JetBrains Marketplace for [installation instructions](https://plugins.jetbrains.com/plugin/9682-sourcegraph)
```

File: doc/cody/index.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ Cody uses a combination of Sourcegraph's code graph and Large Language Models (L
 
 Cody is also available as an editor extension that can be connected to a Sourcegraph enterprise instance, the Cody app, or Sourcegraph.com (for open source code only):
   - [VS Code extension](https://marketplace.visualstudio.com/items?itemName=sourcegraph.cody-ai)
-  - Jetbrains extension (coming soon)
+  - [Jetbrains extension (Experimental)](https://plugins.jetbrains.com/plugin/9682-sourcegraph)
```
Comment (vdavid) [inline doc/cody/index.md:14]:
```suggestion
  - [JetBrains extension (Experimental)](https://plugins.jetbrains.com/plugin/9682-sourcegraph)
```

**Review by vdavid — APPROVED** [review]
Left some suggestions, approving to unblock

**Pull request merged**